Makefile

```diff
@@ -205,14 +206,15 @@ $(DIST) $(INSTALLER):
 	mkdir -p $@
 
 # convenience targets - so you can do `make config` instead of `make dist/config.img`, and `make installer` instead of `make dist/amd64/installer.img
+initrd: $(INITRD_IMG)
 config: $(CONFIG_IMG)
 rootfs: $(ROOTFS_IMG)
 live: $(LIVE_IMG).img
 installer: $(INSTALLER).raw
 installer-iso: $(INSTALLER).iso
 
-$(CONFIG_IMG): conf/server conf/onboard.cert.pem conf/authorized_keys conf/ | $(INSTALLER)
-	./tools/makeconfig.sh $(CONF_DIR) $@
+$(CONFIG_IMG): $(CONF_DIR) FORCE | $(INSTALLER)
```

Comment:
`CONF_DIR` just resolves to `conf/`, whereas previously it had dependencies on 4 things in `conf/`. I also don't see any target for `CONF_DIR`, on which this should depend. How does this get resolved?

Reply:
Also `FORCE` is a dependency, but it doesn't do anything?

```Makefile
FORCE:

```

Reply:
FORCE is a .PHONY target that makes sure that the CONFIG_IMG always gets triggered. The issue here is that it takes very little time to actually rebuild the image so not tracking dependencies felt like the right thing to do (hence dropped dependencies from conf/server, etc.)

In order to actually track these depedencnies we'd have to have somewhat complicated logic of accounting for the fact that some of the files in conf/ actually need to get renamed before they are useful. This trips up barebones make.